praetorian_cli/handlers/hackerone.py

```
@@ -0,0 +1,302 @@
+
+    Retrieves recent activity events from HackerOne. Optionally filter by
+    program handle and cap results with --limit.
+
+    \b
+    Example usages:
+        guard hackerone activities
+        guard hackerone activities --program acme-corp
+        guard hackerone activities --limit 50
+        guard hackerone activities --program acme-corp --limit 10 --json-output
+    """
+    params = f'hackerone/activities'
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)